MyCryptoHQ/MyCrypto
# Show Disabled Send Button on /pushtx

Closes #1291

### Description

This is my first ever pull request. Honestly, I'm not sure if I solved this problem in the best way possible but I am happy to hear feedback and try resolving this issue again in a different way if need be.

Task was to make the send transaction button appear on the /pushTx page while no transaction has been entered yet. The button is to remain disabled until a valid transaction is entered. 

### Changes

The main issue I encountered was that there is one SendButton class that creates all the Send Buttons for the project. Every other time, the button remains invisible until a valid transaction is generated. In order to modify the button to appear while invalid, I had to directly modify the SendButtonFactory class which would've broken the button for the other use cases (e.g. Contract Call). 

* Added "disabled" to the SendButton class which allows the button to be disabled depending on whether a valid transaction has been submitted. 
* To get around breaking the button for other use cases, I added a disabledToggle boolean that allows you to toggle the type of SendButton that you want (default is the existing send button type). 

### Steps to Test

1. Visit /pushTx and you will see that the button starts off disabled. Paste in a valid tx and it will become enabled.
2. Visit /contracts/interact and enter a valid contract/ABI and then attempt to sign a transaction and broadcast it. Button will appear normally as expected.
